test/translator/inquiry_test.cc

```diff
@@ -14,6 +15,8 @@
 
 #include "lib/translator/inquiry.h"
 
+#include <netinet/in.h>
+
```

Comment:
do we need to check kernel here?

Reply:
I think it's reasonable to expect that the tests will be run in the userspace